### PR TITLE
Add a IIIF download option for cropped GeoTiffs

### LINKED ISSUE #5857: Allow download of cropped GeoTiff in UniversalViewer
The maps group would like to see a download of the original GeoTiff as well as the cropped version here: https://maps.princeton.edu/catalog/princeton-dc1257b365d 

The original tiff appears, but the cropped one isn't downloadable.

## Implementation Notes

We'd have to add this as another "Rendering" in the IIIF Manifest.

### PR DESCRIPTION
**Pull request by eliotjordan**
Closes #5857

### CODE REVIEW

File: yarn.lock
Comment (tpendragon) [inline yarn.lock:1]: Why'd yarn.lock change?

File: app/services/manifest_builder.rb [hunk 1]
```diff
@@ -635,6 +635,22 @@ def geotiff_child
         end
     end
 
+    def cropped_geotiff_child
```
Comment (tpendragon) [inline app/services/manifest_builder.rb:638]: Question: Would it make sense to move a bunch of this navigation logic to the wayfinder? So we can do like Wayfinder.for(resource).try(:cropped_geotiff_child)`?

**Review by hackartisan — COMMENTED** [review]
Is the same change also needed in `manifest_builder_v3/canvas_rendering_builder.rb`? I'm not sure I have a great sense of what's serving v2 vs v3 manifests.

File: yarn.lock
Comment (eliotjordan) [reply yarn.lock:1]: Accident. Was probably trying to get node-mapnik installed.

File: app/services/manifest_builder.rb (hunk 1 above)
Comment (eliotjordan) [reply app/services/manifest_builder.rb]: Good idea. Done.

**Review by tpendragon — CHANGES_REQUESTED** [review]
Code looks good, that yarn.lock is just still there.

**Review by tpendragon — APPROVED** [review]

**Pull request merged**